apps/studio.giselles.ai/app/giselle-engine.ts

```diff
@@ -5,15 +5,18 @@ import { WorkspaceId } from "@giselle-sdk/data-type";
 import { NextGiselleEngine } from "@giselle-sdk/giselle-engine/next";
 import { createStorage } from "unstorage";
 import fsDriver from "unstorage/drivers/fs";
-import vercelBlobDriver from "unstorage/drivers/vercel-blob";
+import s3Driver from "unstorage/drivers/s3";
 
 const isVercelEnvironment = process.env.VERCEL === "1";
 
 const storage = createStorage({
 	driver: isVercelEnvironment
-		? vercelBlobDriver({
-				access: "public",
-				base: "private-beta",
+		? s3Driver({
+				accessKeyId: process.env.SUPABASE_ACCESS_KEY_ID ?? "",
+				secretAccessKey: process.env.SUPABASE_SECRET_ACCESS_KEY ?? "",
+				endpoint: process.env.SUPABASE_ENDPOINT ?? "",
+				region: "us-west-1",
```

Comment:
```suggestion
				region: "us-east-1",
```

Reply:
Since we have unified the region of Vercel Functions and Vercel Postgres, Supabase Auth to `us-east-1`, we would like you to change it to `us-east-1` as well from a latency perspective.

Reply:
I updated the suggested change by https://github.com/giselles-ai/giselle/commit/1b9485cc76bdebeb18c30e351f50a517e668910a!
@toyamarinyon